### PR TITLE
Adds craftable skull helmet.

### PR DESCRIPTION
**Pull request by TechnoAlchemisto**
Dirty tribal armor set complete.

:cl:
rscadd: You can now craft skull helmets from bones.
/:cl:

### CODE REVIEW

**Comment by Bawhoppen** [comment]
heojs

**Comment by Iamgoofball** [comment]
we rust now

**Comment by TechnoAlchemisto** [comment]
oonga boonga

**Comment by KorPhaeron** [comment]
I wish we had a way to make all this weird shit like boats not clutter regular crafting menus. Should probably make it have its own crafting category eventually

**Comment by MMMiracles** [comment]
im here for the 'hooray' emote.

**Comment by TechnoAlchemisto** [comment]
It's easy, it's free, it's fun!

File: code/__DEFINES/tablecrafting.dm [hunk 1]
```diff
@@ -4,4 +4,5 @@
 #define CAT_AMMO	"Ammunition"
 #define CAT_ROBOT	"Robots"
 #define CAT_FOOD	"Food"
-#define CAT_MISC	"Misc"
\ No newline at end of file
+#define CAT_MISC	"Misc"
+#define CAT_PRIMAL  "Tribal"
```
Comment (KorPhaeron) [inline code/__DEFINES/tablecrafting.dm]: please save my OCD and make "tribal" line up with the others

Comment (TechnoAlchemisto) [inline code/__DEFINES/tablecrafting.dm]: http://prntscr.com/axcfug I think it's just displaying weird

**Comment by KorPhaeron** [comment]
Moving right along with preserving everyone's sanity regarding the commonly used dmi files

**Pull request merged**